src/ViewHelpers/InlineViewHelper.php

```diff
@@ -0,0 +1,64 @@
+<?php
+namespace TYPO3Fluid\Fluid\ViewHelpers;
+
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+use TYPO3Fluid\Fluid\Core\ViewHelper\Traits\CompileWithContentArgumentAndRenderStatic;
+
+/**
+ * Inline Fluid rendering ViewHelper
+ *
+ * Renders Fluid code stored in a variable, which you normally would
+ * have to render before assigning it to the view. Instead you can
+ * do the following (note, extremely simplified use case):
+ *
+ *      $view->assign('variable', 'value of my variable');
+ *      $view->assign('code', 'My variable: {variable}');
+ *
+ * And in the template:
+ *
+ *      {code -> f:inline()}
+ *
+ * Which outputs:
+ *
+ *      My variable: value of my variable
+ *
+ * You can use this to pass smaller and dynamic pieces of Fluid code
+ * to templates, as an alternative to creating new partial templates.
+ */
+class InlineViewHelper extends AbstractViewHelper
+{
+    use CompileWithContentArgumentAndRenderStatic;
+
+    protected $escapeChildren = false;
+
+    protected $escapeOutput = false;
+
+    /**
+     * @return void
+     */
+    public function initializeArguments()
+    {
+        $this->registerArgument(
+            'code',
+            'string',
+            'Fluid code to be rendered as if it were part of the template rendering it. Can be passed as inline '.
+            'argument or tag content'
```

Comment:
Please merge these two lines. Right now it looks like 4 arguments are passed to `registerArgument()`

Reply:
Just for the record I'm personally all for allowing exceptions of longer lines but technically we do have a 130 char max. Again personally, I think it's time we all started reconsidering this limit because it isn't always a good idea to blindly enforce it ;) at any rate: changed, pushed.